addon/components/frost-fixed-table.js

```diff
@@ -334,5 +338,20 @@ export default Component.extend({
   // == Actions ===============================================================
 
   actions: {
+    /**
+     * Generic action dispatcher, so that cell renderers can indicate arbitrary events.
+     * Your handler is then responsible for doing stuff based on actions like 'click' or 'input'.
+     *
+     * @param {Number} row - data rows are zero based, and header has row -1
+     * @param {Number} col - column index
+     * @param {String} action - this comes after row/col as Ember lets us include those in action closures easily.
+     * @param {Object[]} args - any additional data
+     */
+    genericDispatch (row, col, action, args) {
```

Comment:
Inside the components, it's convenient to have the order be `row`, `col`, then `action`, `args` (in whichever order), because then Ember's closures over actions already encode the row + column before they get to one's renderer.  That way, your custom renderer does not need to know its row + column when invoking.  

However, I didn't really want to have `row, col, action, args` be the order used by our custom callback, because I felt `action, row col, args` was nicer DX.  After talking a bit with @sandersky this morning, I like his suggestion of putting all of them into a single object, so that one's callback takes one parameter (which we can destructure).

Then we'd have,
```javascript
    genericDispatch (row, col, action, args) {
      const handler = this.get('callbackHandler')
      if (handler) {
        handler({action, row, col, args})
      }
    }
```